wkcuber/api/mag_view.py

```diff
@@ -328,6 +328,12 @@ def _get_file_dimensions(self) -> Tuple[int, int, int]:
             Tuple[int, int, int], (self.header.file_len * self.header.block_len,) * 3
         )
 
+    def __repr__(self) -> str:
+        return repr(
+            "MagView(%s, name=%s, global_offset=%s, size=%s)"
+            % (self.path, self.name, self.global_offset, self.size)
```

Comment:
I am not sure if the path is important here. In any case, the name should come first.